callbacks/segmentation.py

```diff
@@ -427,37 +438,53 @@ def populate_segmentation_results(
                     ANNOT_ICONS["results"],
                     f"Retrieved result from {job_type} job!",
                 )
-                return notification, result_store
+
+                return notification, result_store, children_flows[0]
             else:
-                return no_update, None
-    return no_update, no_update
+                return no_update, None, None
+    return no_update, no_update, None
 
 
 @callback(
     Output("notifications-container", "children", allow_duplicate=True),
     Output("seg-results-train-store", "data"),
+    Output("dvc-training-stats-link", "href"),
     Input("train-job-selector", "value"),
-    Input("project-name-src", "value"),
+    State("project-name-src", "value"),
     prevent_initial_call=True,
 )
 def populate_segmentation_results_train(train_job_id, project_name):
     """
     This callback populates the segmentation results store based on the uids
     if the training job and the inference job.
     """
-    return populate_segmentation_results(train_job_id, project_name, "training")
+    notification, result_store, segment_job_id = populate_segmentation_results(
+        train_job_id, project_name, "training"
+    )
+    if segment_job_id is not None:
+        results_link = os.path.join(RESULTS_DIR, segment_job_id, "results.html")
```

Comment:
The current version in [mlex_dlsia](https://github.com/mlexchange/mlex_dlsia_segmentation_prototype) has setup the path to the DVC report to `dvc_metrics/report.html`
I made that change at my end to test it, but I am unable to see the DVC report from the browser. The URL updates to: `http://localhost:8075/path/to/report.html`, with `path/to/report.htm` being the correct path to the report. I wonder if there is a routing issue to the html file.

Reply:
It appears that linking to a local file is not permitted in most browsers due to security concerns and requires to start the browser in a certain way, or use of an extension. We may have to explore other means to show these metrics. 

Copying the same url in the tab, pasting it, then pressing enter should make the report show up though.

Reply:
That makes sense. Just a minor comment here is that the path to the dvc report has been updated in the latest version of dlsia from `results.html` to `dvc_metrics/report.html`, as detailed in the following [link](https://github.com/mlexchange/mlex_dlsia_segmentation_prototype/blob/e76c22afe25facefd490e40ce67647f85aa29474/src/train.py#L118).
```suggestion
        results_link = os.path.join(RESULTS_DIR, segment_job_id, "dvc_metrics/report.html")
```